some.json

```diff
@@ -0,0 +1,13 @@
+{
+  "email": "",
+  "prBoard": "",
+  "comment": {
```

Comment:
/ticket fix this field -d this field is legacy and should be removed

Reply:
Here's your ticket: [fix this field](https://ticketgen19.atlassian.net/browse/IKKI-37)